.github/workflows/ci.yml

```
@@ -204,29 +204,8 @@ jobs:
-          echo "ROCM_PATH=${ROCM_PATH}" >> $GITHUB_ENV
-          echo "CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}" >> $GITHUB_ENV
-          echo "HIP_PLATFORM=${HIP_PLATFORM}" >> $GITHUB_ENV
-          echo "HSA_PATH=${HSA_PATH}" >> $GITHUB_ENV
-          echo "PATH=${PATH}" >> $GITHUB_ENV
-
-          which clang++
-          clang++ --version
-          hipconfig --platform
-          hipconfig -v
+        shell: bash
+        run: $APCI_ALPAKA_ROOT/script/ci/install/rocm.sh
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting

(shellcheck)